backend/editor/OT/operations/tests/operation_application_test.go

```diff
@@ -194,14 +194,14 @@ func TestInsertStringOperation(t *testing.T) {
 
 	jsonOperation := `{
 		"Path": [2, 0, 1],
-		"OperationType": 0,
 		"AcknowledgedServerOps": 0,
 		"IsNoOp": false,
 		"Operation": {
-			"$type": "stringOperation",
+			"type": "stringOperation",
 			"RangeStart": 5,
 			"RangeEnd": 5,
 			"NewValue": "0"
+			"OperationType": 0,
```

Comment:
Don't use explicit integers to refer to elements of an enum

Reply:
done 🚀 skill issues tbh